core/src/trezor/lvglui/scrs/homescreen.py

```
@@ -4747,28 +4809,42 @@ def on_click(self, event_obj):
-                                on_remove=lambda: self.on_remove(select_index),
+                for i, item in enumerate(self.added_fingerprints):
+                    if target == item:
+                        if self.data_new_version:
+                            group = self.groups[i]
+                            prompt = _(i18n_keys.FORM__FINGER_STR).format(
+                                group["group_id"] + 1
+                            )
+                            workflow.spawn(
+                                fingerprints.request_delete_fingerprint(
+                                    prompt,
+                                    on_remove=lambda: self.on_remove_group(group),
```

Ruff (0.8.2)

Function definition does not bind loop variable `group`

(B023)